arti_base_control/OdometryAxelCalculationInfo[] axles
geometry_msgs/Twist odom_velocity
float64 time_difference
geometry_msgs/Pose2D odom_pose
